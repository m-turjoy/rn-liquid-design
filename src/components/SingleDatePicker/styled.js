import styled from 'styled-components';
import {
  backgroundColor,
  width,
  height,
  color,
  fontFamily,
  fontSize,
  paddingLeft,
} from 'styled-system';

const CalendarContainer = styled.View``;

const CalendarWrapper = styled.View``;

const IconLabelTouchable = styled.TouchableHighlight``;

const TouchableModalChildrenWrapper = styled.TouchableWithoutFeedback``;

const TextFieldsWrapper = styled.View``;

const IconWrapper = styled.View``;

const ContainerWrapper = styled.View``;

export {
  CalendarContainer,
  CalendarWrapper,
  IconLabelTouchable,
  TouchableModalChildrenWrapper,
  TextFieldsWrapper,
  IconWrapper,
  ContainerWrapper,
};
