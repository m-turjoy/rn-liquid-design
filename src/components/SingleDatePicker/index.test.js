import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SingleDatePicker from './index';

configure({ adapter: new Adapter() });
describe('SingleDatePicker component', () => {
  describe('Renders correctly', () => {
    test('it renders default Slider', () => {
      const tree = shallow(<SingleDatePicker />);
      expect(tree).toMatchSnapshot();
    });
  });
});
