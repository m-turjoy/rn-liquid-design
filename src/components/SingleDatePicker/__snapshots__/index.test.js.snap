// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleDatePicker component Renders correctly it renders default Slider 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SingleDatePicker
    disabled={false}
    errorMessage="* Error Message"
    firstDay={1}
    labelText="Start date"
    onEndEditing={[Function]}
    themeName="vibrantCyan"
    withLabel={true}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Styled(View)
        flexDirection="column"
      >
        <Styled(View)
          alignItems="flex-start"
          flexDirection="row"
          height={80}
          justifyContent="flex-start"
        >
          <TextField
            backgroundColor="#f8f8fc"
            borderRadius={6}
            bothTypesWrapperStyle={
              Object {
                "paddingLeft": 15,
              }
            }
            color="#1b1b25"
            disabled={false}
            error={false}
            errorMessage="* Error Message"
            errorMessageColor="#e61e50"
            errorMessageFontFamily="Lato-Regular"
            errorMessageFontSize={12}
            errorMessageStyle={Object {}}
            fontFamily="Lato-Regular"
            fontSize={14}
            keyboardType="numeric"
            maxLength={14}
            multiline={false}
            multilineWrapperHeight={200}
            multilineWrapperWidth={300}
            onBlur={[Function]}
            onChange={[Function]}
            onEndEditing={[Function]}
            onFocus={[Function]}
            paddingBottom={6}
            paddingTop={6}
            placeholder="dd / mm / yyyy"
            placeholderTextColor="#93939d"
            textInputLabel="Start date"
            textInputLabelColor="#93939d"
            textInputLabelFontFamily="Lato-Regular"
            textInputLabelFontSize={12}
            textInputLabelStyle={Object {}}
            textInputLabelVisible={true}
            themeName="vibrantCyan"
            wrapperHeight={40}
            wrapperWidth={120}
          />
          <Styled(TouchableHighlight)
            activeOpacity={1}
            onPress={[Function]}
            underlayColor="transparent"
          >
            <Styled(View)
              height={60}
              marginLeft={10}
              top={30}
            >
              <Icon
                color="#2dbecd"
                name="calendar"
                size={20}
                themeName="vibrantCyan"
              />
            </Styled(View)>
          </Styled(TouchableHighlight)>
        </Styled(View)>
      </Styled(View)>,
      "theme": Object {
        "colors": Object {
          "black": Object {
            "base": undefined,
          },
          "primary": Object {
            "base": "#2dbecd",
            "dark": "#29abb9",
            "darker": "#208590",
            "light": "#57cbd7",
            "lightest": "#abe5eb",
          },
          "richBlack": Object {
            "base": undefined,
            "dark": undefined,
            "darker": undefined,
            "darkest": undefined,
            "light": undefined,
            "lighter": undefined,
            "lightest": undefined,
          },
          "richBlue": Object {
            "base": "#0f69af",
            "dark": "#0e5f9e",
            "darker": "#0b4a7b",
            "darkest": "#083558",
            "light": "#3f87bf",
            "lighter": "#6fa5cf",
            "lightest": "#9fc3df",
          },
          "richGreen": Object {
            "base": "#149b5f",
            "dark": "#0d6c42",
            "darker": "#0c5d39",
            "darkest": "#094d2f",
            "light": "#5ab98f",
            "lighter": "#72c39f",
            "lightest": "#89cdaf",
          },
          "richPurple": Object {
            "base": "#503291",
            "dark": "#482d83",
            "darker": "#382366",
            "darkest": "#281949",
            "light": "#735ba7",
            "lighter": "#9684bd",
            "lightest": "#b9add3",
          },
          "richRed": Object {
            "base": "#e61e50",
            "dark": "#cf1b48",
            "darker": "#a11538",
            "darkest": "#730f28",
            "light": "#eb4b73",
            "lighter": "#f07896",
            "lightest": "#f5a5b9",
          },
          "secondary": Object {
            "base": "#ffc832",
            "dark": "#f3be2f",
            "darker": "#ddac28",
            "light": "#ffd35b",
            "lightest": "#ffe9ad",
          },
          "sensitiveGrey": Object {
            "base": "#f8f8fc",
            "dark": "#f3f3f7",
            "darker": "#efeff1",
            "darkest": "#e9e9ed",
            "light": "#f8f8fc",
            "lighter": "#ffffff",
            "lightest": "#ffffff",
          },
          "vibrantCyan": Object {
            "base": "#2dbecd",
            "dark": "#29abb9",
            "darker": "#208590",
            "darkest": "#175f67",
            "light": "#57cbd7",
            "lighter": "#81d8e1",
            "lightest": "#abe5eb",
          },
          "vibrantGreen": Object {
            "base": undefined,
            "dark": undefined,
            "darker": undefined,
            "darkest": undefined,
            "light": undefined,
            "lighter": undefined,
            "lightest": undefined,
          },
          "vibrantMagenta": Object {
            "base": "#eb3c96",
            "dark": "#d43687",
            "darker": "#a52a69",
            "darkest": "#761e4b",
            "light": "#ef63ab",
            "lighter": "#f38ac0",
            "lightest": "#f7b1d5",
          },
          "vibrantYellow": Object {
            "base": "#ffc832",
            "dark": "#f3be2f",
            "darker": "#ddac28",
            "darkest": "#806419",
            "light": "#ffd35b",
            "lighter": "#ffde84",
            "lightest": "#ffe9ad",
          },
          "white": Object {
            "base": "#ffffff",
          },
        },
        "fontSizes": Array [
          8,
          10,
          12,
          14,
          16,
          18,
          20,
          24,
          36,
        ],
        "fontWeights": Array [
          100,
          200,
          300,
          400,
          500,
          600,
          700,
          800,
          900,
        ],
        "fonts": Object {
          "Black": "Lato-Black",
          "Bold": "Lato-Bold",
          "IcoMoon": "icomoon",
          "Merck": "MerckWeb-Regular",
          "Regular": "Lato-Regular",
        },
        "heights": Object {
          "Big": 50,
          "Default": 40,
        },
        "shadows": Object {
          "active": Object {
            "elevation": 40,
            "shadowColor": "#1b1b25",
            "shadowOffset": "0px 30px",
            "shadowOpacity": 0.3,
            "shadowRadius": 20,
          },
          "default": Object {},
          "stacked": Object {
            "elevation": 25,
            "shadowColor": "#1b1b25",
            "shadowOffset": "0px 30px",
            "shadowOpacity": 0.3,
            "shadowRadius": 20,
          },
        },
        "space": Array [
          0,
          1,
          2,
          5,
          10,
          15,
          20,
          24,
          50,
          8,
          7,
          4,
          9,
          6,
        ],
        "stacked": Object {
          "stackedFirst": Object {
            "borderBottomWidth": 1,
            "borderColor": "#f3f3f7",
            "borderRightWidth": 1,
            "position": "absolute",
            "zIndex": 10,
          },
          "stackedSecond": Object {
            "borderBottomWidth": 1,
            "borderColor": "#f3f3f7",
            "borderRightWidth": 1,
            "left": 3,
            "position": "absolute",
            "top": 3,
            "zIndex": 9,
          },
          "stackedThird": Object {
            "borderBottomWidth": 1,
            "borderColor": "#f3f3f7",
            "borderRightWidth": 1,
            "left": 6,
            "top": 6,
            "zIndex": 8,
          },
        },
        "widths": Array [
          50,
          100,
          130,
        ],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Styled(View)
            alignItems="flex-start"
            flexDirection="row"
            height={80}
            justifyContent="flex-start"
          >
            <TextField
              backgroundColor="#f8f8fc"
              borderRadius={6}
              bothTypesWrapperStyle={
                Object {
                  "paddingLeft": 15,
                }
              }
              color="#1b1b25"
              disabled={false}
              error={false}
              errorMessage="* Error Message"
              errorMessageColor="#e61e50"
              errorMessageFontFamily="Lato-Regular"
              errorMessageFontSize={12}
              errorMessageStyle={Object {}}
              fontFamily="Lato-Regular"
              fontSize={14}
              keyboardType="numeric"
              maxLength={14}
              multiline={false}
              multilineWrapperHeight={200}
              multilineWrapperWidth={300}
              onBlur={[Function]}
              onChange={[Function]}
              onEndEditing={[Function]}
              onFocus={[Function]}
              paddingBottom={6}
              paddingTop={6}
              placeholder="dd / mm / yyyy"
              placeholderTextColor="#93939d"
              textInputLabel="Start date"
              textInputLabelColor="#93939d"
              textInputLabelFontFamily="Lato-Regular"
              textInputLabelFontSize={12}
              textInputLabelStyle={Object {}}
              textInputLabelVisible={true}
              themeName="vibrantCyan"
              wrapperHeight={40}
              wrapperWidth={120}
            />
            <Styled(TouchableHighlight)
              activeOpacity={1}
              onPress={[Function]}
              underlayColor="transparent"
            >
              <Styled(View)
                height={60}
                marginLeft={10}
                top={30}
              >
                <Icon
                  color="#2dbecd"
                  name="calendar"
                  size={20}
                  themeName="vibrantCyan"
                />
              </Styled(View)>
            </Styled(TouchableHighlight)>
          </Styled(View)>,
          null,
        ],
        "flexDirection": "column",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "alignItems": "flex-start",
            "children": Array [
              <TextField
                backgroundColor="#f8f8fc"
                borderRadius={6}
                bothTypesWrapperStyle={
                  Object {
                    "paddingLeft": 15,
                  }
                }
                color="#1b1b25"
                disabled={false}
                error={false}
                errorMessage="* Error Message"
                errorMessageColor="#e61e50"
                errorMessageFontFamily="Lato-Regular"
                errorMessageFontSize={12}
                errorMessageStyle={Object {}}
                fontFamily="Lato-Regular"
                fontSize={14}
                keyboardType="numeric"
                maxLength={14}
                multiline={false}
                multilineWrapperHeight={200}
                multilineWrapperWidth={300}
                onBlur={[Function]}
                onChange={[Function]}
                onEndEditing={[Function]}
                onFocus={[Function]}
                paddingBottom={6}
                paddingTop={6}
                placeholder="dd / mm / yyyy"
                placeholderTextColor="#93939d"
                textInputLabel="Start date"
                textInputLabelColor="#93939d"
                textInputLabelFontFamily="Lato-Regular"
                textInputLabelFontSize={12}
                textInputLabelStyle={Object {}}
                textInputLabelVisible={true}
                themeName="vibrantCyan"
                wrapperHeight={40}
                wrapperWidth={120}
              />,
              <Styled(TouchableHighlight)
                activeOpacity={1}
                onPress={[Function]}
                underlayColor="transparent"
              >
                <Styled(View)
                  height={60}
                  marginLeft={10}
                  top={30}
                >
                  <Icon
                    color="#2dbecd"
                    name="calendar"
                    size={20}
                    themeName="vibrantCyan"
                  />
                </Styled(View)>
              </Styled(TouchableHighlight)>,
            ],
            "flexDirection": "row",
            "height": 80,
            "justifyContent": "flex-start",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "backgroundColor": "#f8f8fc",
                "borderRadius": 6,
                "bothTypesWrapperStyle": Object {
                  "paddingLeft": 15,
                },
                "color": "#1b1b25",
                "disabled": false,
                "error": false,
                "errorMessage": "* Error Message",
                "errorMessageColor": "#e61e50",
                "errorMessageFontFamily": "Lato-Regular",
                "errorMessageFontSize": 12,
                "errorMessageStyle": Object {},
                "fontFamily": "Lato-Regular",
                "fontSize": 14,
                "keyboardType": "numeric",
                "maxLength": 14,
                "multiline": false,
                "multilineWrapperHeight": 200,
                "multilineWrapperWidth": 300,
                "onBlur": [Function],
                "onChange": [Function],
                "onEndEditing": [Function],
                "onFocus": [Function],
                "paddingBottom": 6,
                "paddingTop": 6,
                "placeholder": "dd / mm / yyyy",
                "placeholderTextColor": "#93939d",
                "textInputLabel": "Start date",
                "textInputLabelColor": "#93939d",
                "textInputLabelFontFamily": "Lato-Regular",
                "textInputLabelFontSize": 12,
                "textInputLabelStyle": Object {},
                "textInputLabelVisible": true,
                "themeName": "vibrantCyan",
                "value": undefined,
                "wrapperHeight": 40,
                "wrapperWidth": 120,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeOpacity": 1,
                "children": <Styled(View)
                  height={60}
                  marginLeft={10}
                  top={30}
                >
                  <Icon
                    color="#2dbecd"
                    name="calendar"
                    size={20}
                    themeName="vibrantCyan"
                  />
                </Styled(View)>,
                "onPress": [Function],
                "underlayColor": "transparent",
              },
              "ref": [Function],
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Icon
                    color="#2dbecd"
                    name="calendar"
                    size={20}
                    themeName="vibrantCyan"
                  />,
                  "height": 60,
                  "marginLeft": 10,
                  "top": 30,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "color": "#2dbecd",
                    "name": "calendar",
                    "size": 20,
                    "themeName": "vibrantCyan",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        null,
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(View)
          flexDirection="column"
        >
          <Styled(View)
            alignItems="flex-start"
            flexDirection="row"
            height={80}
            justifyContent="flex-start"
          >
            <TextField
              backgroundColor="#f8f8fc"
              borderRadius={6}
              bothTypesWrapperStyle={
                Object {
                  "paddingLeft": 15,
                }
              }
              color="#1b1b25"
              disabled={false}
              error={false}
              errorMessage="* Error Message"
              errorMessageColor="#e61e50"
              errorMessageFontFamily="Lato-Regular"
              errorMessageFontSize={12}
              errorMessageStyle={Object {}}
              fontFamily="Lato-Regular"
              fontSize={14}
              keyboardType="numeric"
              maxLength={14}
              multiline={false}
              multilineWrapperHeight={200}
              multilineWrapperWidth={300}
              onBlur={[Function]}
              onChange={[Function]}
              onEndEditing={[Function]}
              onFocus={[Function]}
              paddingBottom={6}
              paddingTop={6}
              placeholder="dd / mm / yyyy"
              placeholderTextColor="#93939d"
              textInputLabel="Start date"
              textInputLabelColor="#93939d"
              textInputLabelFontFamily="Lato-Regular"
              textInputLabelFontSize={12}
              textInputLabelStyle={Object {}}
              textInputLabelVisible={true}
              themeName="vibrantCyan"
              wrapperHeight={40}
              wrapperWidth={120}
            />
            <Styled(TouchableHighlight)
              activeOpacity={1}
              onPress={[Function]}
              underlayColor="transparent"
            >
              <Styled(View)
                height={60}
                marginLeft={10}
                top={30}
              >
                <Icon
                  color="#2dbecd"
                  name="calendar"
                  size={20}
                  themeName="vibrantCyan"
                />
              </Styled(View)>
            </Styled(TouchableHighlight)>
          </Styled(View)>
        </Styled(View)>,
        "theme": Object {
          "colors": Object {
            "black": Object {
              "base": undefined,
            },
            "primary": Object {
              "base": "#2dbecd",
              "dark": "#29abb9",
              "darker": "#208590",
              "light": "#57cbd7",
              "lightest": "#abe5eb",
            },
            "richBlack": Object {
              "base": undefined,
              "dark": undefined,
              "darker": undefined,
              "darkest": undefined,
              "light": undefined,
              "lighter": undefined,
              "lightest": undefined,
            },
            "richBlue": Object {
              "base": "#0f69af",
              "dark": "#0e5f9e",
              "darker": "#0b4a7b",
              "darkest": "#083558",
              "light": "#3f87bf",
              "lighter": "#6fa5cf",
              "lightest": "#9fc3df",
            },
            "richGreen": Object {
              "base": "#149b5f",
              "dark": "#0d6c42",
              "darker": "#0c5d39",
              "darkest": "#094d2f",
              "light": "#5ab98f",
              "lighter": "#72c39f",
              "lightest": "#89cdaf",
            },
            "richPurple": Object {
              "base": "#503291",
              "dark": "#482d83",
              "darker": "#382366",
              "darkest": "#281949",
              "light": "#735ba7",
              "lighter": "#9684bd",
              "lightest": "#b9add3",
            },
            "richRed": Object {
              "base": "#e61e50",
              "dark": "#cf1b48",
              "darker": "#a11538",
              "darkest": "#730f28",
              "light": "#eb4b73",
              "lighter": "#f07896",
              "lightest": "#f5a5b9",
            },
            "secondary": Object {
              "base": "#ffc832",
              "dark": "#f3be2f",
              "darker": "#ddac28",
              "light": "#ffd35b",
              "lightest": "#ffe9ad",
            },
            "sensitiveGrey": Object {
              "base": "#f8f8fc",
              "dark": "#f3f3f7",
              "darker": "#efeff1",
              "darkest": "#e9e9ed",
              "light": "#f8f8fc",
              "lighter": "#ffffff",
              "lightest": "#ffffff",
            },
            "vibrantCyan": Object {
              "base": "#2dbecd",
              "dark": "#29abb9",
              "darker": "#208590",
              "darkest": "#175f67",
              "light": "#57cbd7",
              "lighter": "#81d8e1",
              "lightest": "#abe5eb",
            },
            "vibrantGreen": Object {
              "base": undefined,
              "dark": undefined,
              "darker": undefined,
              "darkest": undefined,
              "light": undefined,
              "lighter": undefined,
              "lightest": undefined,
            },
            "vibrantMagenta": Object {
              "base": "#eb3c96",
              "dark": "#d43687",
              "darker": "#a52a69",
              "darkest": "#761e4b",
              "light": "#ef63ab",
              "lighter": "#f38ac0",
              "lightest": "#f7b1d5",
            },
            "vibrantYellow": Object {
              "base": "#ffc832",
              "dark": "#f3be2f",
              "darker": "#ddac28",
              "darkest": "#806419",
              "light": "#ffd35b",
              "lighter": "#ffde84",
              "lightest": "#ffe9ad",
            },
            "white": Object {
              "base": "#ffffff",
            },
          },
          "fontSizes": Array [
            8,
            10,
            12,
            14,
            16,
            18,
            20,
            24,
            36,
          ],
          "fontWeights": Array [
            100,
            200,
            300,
            400,
            500,
            600,
            700,
            800,
            900,
          ],
          "fonts": Object {
            "Black": "Lato-Black",
            "Bold": "Lato-Bold",
            "IcoMoon": "icomoon",
            "Merck": "MerckWeb-Regular",
            "Regular": "Lato-Regular",
          },
          "heights": Object {
            "Big": 50,
            "Default": 40,
          },
          "shadows": Object {
            "active": Object {
              "elevation": 40,
              "shadowColor": "#1b1b25",
              "shadowOffset": "0px 30px",
              "shadowOpacity": 0.3,
              "shadowRadius": 20,
            },
            "default": Object {},
            "stacked": Object {
              "elevation": 25,
              "shadowColor": "#1b1b25",
              "shadowOffset": "0px 30px",
              "shadowOpacity": 0.3,
              "shadowRadius": 20,
            },
          },
          "space": Array [
            0,
            1,
            2,
            5,
            10,
            15,
            20,
            24,
            50,
            8,
            7,
            4,
            9,
            6,
          ],
          "stacked": Object {
            "stackedFirst": Object {
              "borderBottomWidth": 1,
              "borderColor": "#f3f3f7",
              "borderRightWidth": 1,
              "position": "absolute",
              "zIndex": 10,
            },
            "stackedSecond": Object {
              "borderBottomWidth": 1,
              "borderColor": "#f3f3f7",
              "borderRightWidth": 1,
              "left": 3,
              "position": "absolute",
              "top": 3,
              "zIndex": 9,
            },
            "stackedThird": Object {
              "borderBottomWidth": 1,
              "borderColor": "#f3f3f7",
              "borderRightWidth": 1,
              "left": 6,
              "top": 6,
              "zIndex": 8,
            },
          },
          "widths": Array [
            50,
            100,
            130,
          ],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Styled(View)
              alignItems="flex-start"
              flexDirection="row"
              height={80}
              justifyContent="flex-start"
            >
              <TextField
                backgroundColor="#f8f8fc"
                borderRadius={6}
                bothTypesWrapperStyle={
                  Object {
                    "paddingLeft": 15,
                  }
                }
                color="#1b1b25"
                disabled={false}
                error={false}
                errorMessage="* Error Message"
                errorMessageColor="#e61e50"
                errorMessageFontFamily="Lato-Regular"
                errorMessageFontSize={12}
                errorMessageStyle={Object {}}
                fontFamily="Lato-Regular"
                fontSize={14}
                keyboardType="numeric"
                maxLength={14}
                multiline={false}
                multilineWrapperHeight={200}
                multilineWrapperWidth={300}
                onBlur={[Function]}
                onChange={[Function]}
                onEndEditing={[Function]}
                onFocus={[Function]}
                paddingBottom={6}
                paddingTop={6}
                placeholder="dd / mm / yyyy"
                placeholderTextColor="#93939d"
                textInputLabel="Start date"
                textInputLabelColor="#93939d"
                textInputLabelFontFamily="Lato-Regular"
                textInputLabelFontSize={12}
                textInputLabelStyle={Object {}}
                textInputLabelVisible={true}
                themeName="vibrantCyan"
                wrapperHeight={40}
                wrapperWidth={120}
              />
              <Styled(TouchableHighlight)
                activeOpacity={1}
                onPress={[Function]}
                underlayColor="transparent"
              >
                <Styled(View)
                  height={60}
                  marginLeft={10}
                  top={30}
                >
                  <Icon
                    color="#2dbecd"
                    name="calendar"
                    size={20}
                    themeName="vibrantCyan"
                  />
                </Styled(View)>
              </Styled(TouchableHighlight)>
            </Styled(View)>,
            null,
          ],
          "flexDirection": "column",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "alignItems": "flex-start",
              "children": Array [
                <TextField
                  backgroundColor="#f8f8fc"
                  borderRadius={6}
                  bothTypesWrapperStyle={
                    Object {
                      "paddingLeft": 15,
                    }
                  }
                  color="#1b1b25"
                  disabled={false}
                  error={false}
                  errorMessage="* Error Message"
                  errorMessageColor="#e61e50"
                  errorMessageFontFamily="Lato-Regular"
                  errorMessageFontSize={12}
                  errorMessageStyle={Object {}}
                  fontFamily="Lato-Regular"
                  fontSize={14}
                  keyboardType="numeric"
                  maxLength={14}
                  multiline={false}
                  multilineWrapperHeight={200}
                  multilineWrapperWidth={300}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onEndEditing={[Function]}
                  onFocus={[Function]}
                  paddingBottom={6}
                  paddingTop={6}
                  placeholder="dd / mm / yyyy"
                  placeholderTextColor="#93939d"
                  textInputLabel="Start date"
                  textInputLabelColor="#93939d"
                  textInputLabelFontFamily="Lato-Regular"
                  textInputLabelFontSize={12}
                  textInputLabelStyle={Object {}}
                  textInputLabelVisible={true}
                  themeName="vibrantCyan"
                  wrapperHeight={40}
                  wrapperWidth={120}
                />,
                <Styled(TouchableHighlight)
                  activeOpacity={1}
                  onPress={[Function]}
                  underlayColor="transparent"
                >
                  <Styled(View)
                    height={60}
                    marginLeft={10}
                    top={30}
                  >
                    <Icon
                      color="#2dbecd"
                      name="calendar"
                      size={20}
                      themeName="vibrantCyan"
                    />
                  </Styled(View)>
                </Styled(TouchableHighlight)>,
              ],
              "flexDirection": "row",
              "height": 80,
              "justifyContent": "flex-start",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "backgroundColor": "#f8f8fc",
                  "borderRadius": 6,
                  "bothTypesWrapperStyle": Object {
                    "paddingLeft": 15,
                  },
                  "color": "#1b1b25",
                  "disabled": false,
                  "error": false,
                  "errorMessage": "* Error Message",
                  "errorMessageColor": "#e61e50",
                  "errorMessageFontFamily": "Lato-Regular",
                  "errorMessageFontSize": 12,
                  "errorMessageStyle": Object {},
                  "fontFamily": "Lato-Regular",
                  "fontSize": 14,
                  "keyboardType": "numeric",
                  "maxLength": 14,
                  "multiline": false,
                  "multilineWrapperHeight": 200,
                  "multilineWrapperWidth": 300,
                  "onBlur": [Function],
                  "onChange": [Function],
                  "onEndEditing": [Function],
                  "onFocus": [Function],
                  "paddingBottom": 6,
                  "paddingTop": 6,
                  "placeholder": "dd / mm / yyyy",
                  "placeholderTextColor": "#93939d",
                  "textInputLabel": "Start date",
                  "textInputLabelColor": "#93939d",
                  "textInputLabelFontFamily": "Lato-Regular",
                  "textInputLabelFontSize": 12,
                  "textInputLabelStyle": Object {},
                  "textInputLabelVisible": true,
                  "themeName": "vibrantCyan",
                  "value": undefined,
                  "wrapperHeight": 40,
                  "wrapperWidth": 120,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "activeOpacity": 1,
                  "children": <Styled(View)
                    height={60}
                    marginLeft={10}
                    top={30}
                  >
                    <Icon
                      color="#2dbecd"
                      name="calendar"
                      size={20}
                      themeName="vibrantCyan"
                    />
                  </Styled(View)>,
                  "onPress": [Function],
                  "underlayColor": "transparent",
                },
                "ref": [Function],
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Icon
                      color="#2dbecd"
                      name="calendar"
                      size={20}
                      themeName="vibrantCyan"
                    />,
                    "height": 60,
                    "marginLeft": 10,
                    "top": 30,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "color": "#2dbecd",
                      "name": "calendar",
                      "size": 20,
                      "themeName": "vibrantCyan",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": [Function],
                },
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          null,
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
